docs/components/segmented-control.md

```diff
@@ -118,3 +180,61 @@ export default defineComponent({
 })
 </script>
 ```
+
+## Theming
+
+| Variable | Purpose
+|:-------- |:-------
+| `--KSegmentText` | Option text color
+| `--KSegmentSelectedBackground` | Option background color when selected
+| `--KSegmentSelectedBorder` | Option border color when selected
+| `--KSegmentUnselectedBackground` | Option background color when not selected
+| `--KSegmentUnselectedBorder` | Option border color when not selected
+| `--KSegmentGap` | Gap between the options
+
+An Example of changing the KSegmentedControl to a purple theme instead of blue might look like:
+
+<KComponent :data="{ selected: 'Like it' }" v-slot="{ data }">
+  <KSegmentedControl
+      v-model="data.selected"
+      :options="['Like it', 'Theme it', 'Love it']"
+      class="purple-segment"
+      @click="x => data.selected = x"
+    />
+</KComponent>
+
+```html
+<template>
+  <KComponent :data="{ selected: 'Like it' }" v-slot="{ data }">
+    <KSegmentedControl
+      v-model="data.selected"
+      :options="['Like it', 'Theme it', 'Love it']"
+      class="purple-segment"
+      @click="x => data.selected = x"
+    />
+  </KComponent>
+</template>
+
+<style>
+.purple-segment {
+  --KSegmentText: var(--purple-400);
+  --KSegmentSelectedBackground: var(--white);
+  --KSegmentSelectedBorder: var(--purple-300);
+  --KSegmentUnselectedBackground: var(--blue-100);
+  --KSegmentUnselectedBorder: var(--purple-200);
+  --KSegmentGap: 12px;
```

Comment:
So I thought about this a bit more... the rest of our components utilize CSS variables that are the name of the component, e.g. `--KButtonPrimaryHover`

Can we update the variables used in this PR to be prefixed with `--KSegmentedControl*` ?

Everything else lgtm